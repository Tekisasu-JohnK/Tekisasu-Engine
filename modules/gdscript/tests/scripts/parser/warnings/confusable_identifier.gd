func test():
	var port = 0 # Only latin characters.
	var pοrt = 1 # The "ο" is Greek omicron.

	prints(port, pοrt)
