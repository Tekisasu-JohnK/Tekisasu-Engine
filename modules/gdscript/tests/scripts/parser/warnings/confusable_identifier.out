GDTEST_OK
>> WARNING
>> Line: 3
>> CONFUSABLE_IDENTIFIER
>> The identifier "pÎ¿rt" has misleading characters and might be confused with something else.
0 1
